src/Mac.cpp

```diff
@@ -228,6 +229,22 @@ void Mac::lua(lua_State* vm, bool show_details, bool asListElement) {
 
   lua_push_uint64_table_entry(vm, "pool", get_host_pool());
 
+  lua_push_uint64_table_entry(vm, "talkers.asClient", getNumTalkerAsClient());
```

Comment:
don't clobber the lua of the Mac with all this stuff. Just push non-zero entries only when necessary, that is, when the arp matrix is enabled.